README.md

```diff
@@ -11,7 +11,7 @@ Comes with a naming system, for developerability.
 
 Written in Erlang.
 
-Please see the latest [release notes](RELEASE-NOTES.md) if you want to run the software or just [follow the progress of the project in Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2124891).
+Please see the latest [release notes](docs/release-notes) if you want to run the software or just [follow the progress of the project in Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2124891).
```

Comment:
Let's use either relative (or prefer) or absolute paths. You're using absolute path in following change and relative here.